tsconfig.nocomments

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "declaration": false,
```

Comment:
To clarify, `declaration` to `false` means we won't regenerate our typescript d.ts files, meaning that we won't overwrite those without comments, right?

Reply:
Yes, you're precisely right. 
More than a two-step compilation, it is a 'compile-twice' approach.

Reply:
I was also looking into adding comments somehow, but JSON doesn't officially support comments, so I worry about breaking something if I attempt something brittle like just adding a "comment:" field to the JSON and hoping nothing tries to parse it.

What we can do is create a separate `npm script` called something like `removeComments` and replace the changes I made to the `build` definition to simply call on that other step. 

That way, we attach a proper name to this new step.